roda-core/roda-core/src/main/java/org/roda/core/plugins/base/notifications/EmailIngestNotification.java

```diff
@@ -94,13 +95,16 @@ public void notify(ModelService model, IndexService index, Job job, JobStats job
             StringBuilder builder = new StringBuilder();
 
             for (IndexedReport report : reports) {
-              Report last = report.getReports().get(report.getReports().size() - 1);
+              List<Report> reportsList = RodaCoreFactory.getModelService().retrieveJobReport(report.getJobId(), report.getId()).getReports();
+              Report last = reportsList.get(reportsList.size() - 1);
```

Comment:
Before doing this, ensure that the list is not empty, otherwise the error will occur again.